install_sct.bat

```diff
@@ -136,25 +136,26 @@ if not %SCT_DIR%==%SCT_SOURCE% (
   echo ### Skipping copy of source files ^(source and destination folders are the same^)
 )
 
-rem Move into the SCT installation directory
-pushd %SCT_DIR%
-
-rem Clean old install setup if existing
-if exist bin\ (
+rem Clean old install setup in bin/ if existing
+if exist %SCT_DIR%\bin\ (
   echo ### Removing sct and isct softlink inside the SCT directory...
-  rmdir /s /q bin\ || goto error
+  del %SCT_DIR%\bin\sct_* || goto error
+  del %SCT_DIR%\bin\isct_* || goto error

```

Comment:
What makes Windows special here? Do we expect the `bin\` folder to contain other files that we'd like to keep? Also, if the folder exists but doesn't contain any files starting with `sct_` or `isct_`, will these commands `goto error`?

Reply:
> What makes Windows special here? 

Deleting only `sct_*` and `isct_*` files was actually originally from `install_sct`:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/a485c364ed40beca67942b72cc07f51d869d6d94/install_sct#L532-L536

> Do we expect the bin\ folder to contain other files that we'd like to keep? 

Likely not? _Maybe_ if `SCT_DIR` is accidentally chosen to be an existing non-empty folder, and _maybe_ if there's an existing `bin/` folder, then this would ensure that only the necessary files are deleted.

But, I agree that this feels overly cautious. If anything, I think it would be more beneficial to double-check whether the user actually wants to install SCT to a non-empty directory (by providing a Y/N prompt), which I think would do a lot more to guarantee that no user files are inadvertently deleted.

> Also, if the folder exists but doesn't contain any files starting with `sct_` or `isct_`, will these commands `goto error`?

Good question!

```console
C:\Users\joshu\repos\spinalcordtoolbox>del bin\isct_*
Could Not Find C:\Users\joshu\repos\spinalcordtoolbox\bin\isct_*

C:\Users\joshu\repos\spinalcordtoolbox>echo %errorlevel%
0
```

It would appear not. :slightly_smiling_face:

Reply:
Ah ok, I missed that this was also the behaviour of `install_sct`, in which case it's a separate issue, out of scope for this PR.

And thanks for checking about `|| goto error`!